packages/neo-one-build-tests/tsconfig.jest.json

```diff
@@ -27,10 +27,5 @@
       "bn.js": ["node_modules/@neo-one/build-tools/types/bn.js.d.ts"]
     }
   },
-  "exclude": [
-    "./packages/*/node_modules",
-    "./packages/*/lib",
-    "./packages/*/src/__data__/contracts/**/*.ts",
-    "./packages/*/template/**/*.ts?(x)"
-  ]
+  "exclude": ["./packages/*/src/__data__/contracts/**/*.ts", "./packages/*/template/**/*.ts?(x)"]
```

Comment:
Curious why we wouldn't still need to exclude `lib` and subdirectory `node_modules`? It looks like tsconfig only excludes the root-level `node_modules` by default